example_test.go

```diff
@@ -163,10 +163,10 @@ func ExamplePingRequest_Context() {
 	// Ping a Tarantool instance to check connection.
 	data, err := conn.Do(req).Get()
 	fmt.Println("Ping Resp data", data)
-	fmt.Println("Ping Error", err)
+	fmt.Println("Ping Error", contextDoneErrRegexp.Match([]byte(err.Error())))
```

Comment:
I understand that this is code duplication. But it makes the examples independent from the test code, which I would prefer.
```suggestion
	fmt.Println("Ping Error", regexp.Match(`^context is done \(request ID [0-9]+\)$`, []byte(err.Error())))
```

Reply:
Fixed: I used here a little bit different way